packages/sit-onyx/src/components/OnyxForm/OnyxForm.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import { useDensity } from "../../composables/density";
+import { provideFormContext } from "./OnyxForm.core";
+import type { OnyxFormProps } from "./types";
+
+const props = defineProps<OnyxFormProps>();
```

Comment:
Should we set some mostly used defaults? Like method "dialog"?

Reply:
out of scope of this PR